Tasks/VsTest/helpers.ts

```diff
@@ -118,11 +118,32 @@ export class Helper {
         }
     }
 
-    public static printMultiLineLog (multiLineString : string, logFunction : Function) {
+    public static printMultiLineLog(multiLineString: string, logFunction: Function) {
         const lines = multiLineString.toString().split('\n');
-            lines.forEach(function (line: string) {
-                logFunction(line);
-            });
+        lines.forEach(function (line: string) {
+            logFunction(line);
+        });
+    }
+
+    public static modifyArgument(argument: string): string {
```

Comment:
it wont add quotes again. check the condition. added a test case for the same

---
In reply to: [117964774](https://github.com/Microsoft/vsts-tasks/pull/4383#discussion_r117964774) [](ancestors = 117964774)